shared/actions/teams.tsx

```diff
@@ -1121,7 +1121,7 @@ const deleteChannelConfirmed = async (state: TypedState, action: TeamsGen.Delete
 }
 
 const deleteMultiChannelsConfirmed = async (
-  state: TypedState,
```

Comment:
you should use `chainAction` instead of `chainAction2` here and then you can take out the state parameter